environment_setup/setup_software.sh

```diff
@@ -100,6 +100,7 @@ host_software_packages=(
     protobuf-compiler
     libprotobuf-dev
     libusb-1.0-0-dev
+    qt5-default # The GUI library for our visualizer
```

Comment:
Could we add qt to our `WORKSPACE`? Otherwise we should at least file a ticket to do so.

Reply:
Created #835 will resolve separately